test-lib.sh

```diff
@@ -488,7 +501,15 @@ EOF
     fi
     # Filter out env var definitions from $s2i_args and create Dockerfile ENV commands out of them
     echo "$s2i_args" | grep -o -e '\(-e\|--env\)[[:space:]=]\S*=\S*' | sed -e 's/-e /ENV /' -e 's/--env[ =]/ENV /' >>"$df_name"
+    # Add in artifacts if doing an incremental build
+    if [ $incremental == true ]; then
+        echo "COPY artifacts.tar /tmp/" >>"$df_name"
```

Comment:
Dockerfile ADD instruction could be used to copy in and extract the .tar archive. But have no idea if podman supports this.

Reply:
Thanks for the idea. I expect podman supports this.I tried to stay as close to the multistage Dockerfile from `s2i build --as-dockerfile` [output](https://github.com/sclorg/s2i-nodejs-container/pull/201#issuecomment-462318613) as possible but their use of COPY might have been just because of ADD not being able to work with multistage Dockerfiles.

Let me try it with ADD.